<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry"><rect><x>0</x><y>0</y><width>600</width><height>450</height></rect></property>
  <property name="windowTitle"><string>WeatherApp</string></property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item><widget class="QLineEdit" name="cityLineEdit"/></item>
    <item><widget class="QComboBox" name="rangeComboBox">
      <item><property name="text"><string>Сегодня</string></property></item>
      <item><property name="text"><string>Завтра</string></property></item>
      <item><property name="text"><string>3 дня</string></property></item>
    </widget></item>
    <item><widget class="QPushButton" name="getWeatherBtn">
      <property name="text"><string>Показать прогноз</string></property>
    </widget></item>
    <item><widget class="QTextEdit" name="resultTextEdit"/></item>
    <item><widget class="QLabel" name="historyLabel">
      <property name="text"><string>История:</string></property>
    </widget></item>
    <item><widget class="QTextEdit" name="historyTextEdit">
      <property name="readOnly"><bool>true</bool></property>
    </widget></item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
